Dockerfile

```diff
@@ -59,4 +53,12 @@ RUN echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> /root/.bashrc
 RUN echo 'alias colcon_build="colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release && source install/setup.bash"' >> /root/.bashrc
 
 # Enter bash for development
-CMD ["bash"]
\ No newline at end of file
+CMD ["bash"]
+
+# -----------------------------------------------------------------------
+
+# Note: Ideally this runtime stage should inherit from base and only copy relevant artifacts to it
```

Comment:
Good idea, I'll push a commit that implements this. Tested and working, feel free to polish.

Reply:
Great